frontend/public/module/k8s/types.ts

```diff
@@ -326,6 +326,26 @@ export type DeploymentKind = {
   };
 } & K8sResourceCommon;
 
+export type AppliedClusterResourceQuotaKind = {
+  spec?: {
+    selector?: MatchLabels | Selector;
```

Comment:
This doesn't match the API:
```suggestion
    selector?: {
      labels?: Selector;
      annotations?: MatchLabels;
    };
```